docs/source/getting_started/content/proc_configuring-the-virtual-machine.adoc

```diff
@@ -0,0 +1,50 @@
+[id="configuring-the-virtual-machine_{context}"]
+= Configuring the virtual machine
+
+The default number of vCPUs and amount of memory available to the {prod} virtual machine can be configured using the `cpus` and `memory` configurable properties, respectively.
+
+[IMPORTANT]
+====
+Changes to the configuration of the virtual machine require the creation of a new {prod} virtual machine.
+To create the new virtual machine, first delete the existing {prod} instance, then start a new instance with the configuration changes:
```

Comment:
Is an instance the same as a vm? If so, is there a reason to use a different word here (instance vs vm)?

Reply:
This discussion has been had before... CRC instance is essential the VM. We already had document-wide change for this before.

Orginally we avoided the term VM as this was related to how some companies have a policy against the use of a hypervisor. We have since changed our thinking.

Reply:
I'm just saying that as a reader, I don't intuitively understand that "instance" = "vm", so that's confusing.

Reply:
@robin-owen @stoobie WDYT?

Reply:
@gbraad Better.

Reply:
@gbraad I'm inclined to go with "virtual machine", as that's unambiguous.

Like some other things in this review, that would require changes across the rest of the docs set. I don't have a problem with that, but would want to create a separate issue to address those changes in the near future.

Reply:
The problem came from the use when we talk about both the VM and cluster. the consensus is to use "virtual machine" when talking about the means, but we are starting an "OpenShift cluster" (and for this we use a virtual machine". Perhaps this is why "instance" avoids having to choose between these two? :-/